spec/cucumber/glue/step_definition_spec.rb

```diff
@@ -191,11 +191,12 @@ def step_match(text)
           run_step 'Loud'
         end
 
-        it 'raises an exception if the message is not a String' do
+        it 'calls `to_s` if the message is not a String' do
+          expect(user_interface).to receive(:attach).with('["Not", 1, "string"]', 'text/x.cucumber.log+plain')
           dsl.Given(/Loud/) do
             log ['Not', 1, 'string']
           end
-          expect { run_step 'Loud' }.to raise_exception(Cucumber::LogTypeInvalid)
```

Comment:
Is `Cucumber::LogTypeInvalid` still used? Should it be removed?

Reply:
indeed